doc/source/whatsnew/v2.0.0.rst

```diff
@@ -775,6 +775,7 @@ Performance improvements
 - Performance improvement when iterating over pyarrow and nullable dtypes (:issue:`49825`, :issue:`49851`)
 - Performance improvements to :func:`read_sas` (:issue:`47403`, :issue:`47405`, :issue:`47656`, :issue:`48502`)
 - Memory improvement in :meth:`RangeIndex.sort_values` (:issue:`48801`)
+- Performance improvement in :meth:`Series.to_numpy` if ``copy=True`` but data was copied anyway (:issue:`24345`)
```

Comment:
`data was copied anyway` is kinda confusing since I would expect `copy=True` to have made a copy of the data. Is this supposed to refer to the fact that the data is no longer copied twice?

Reply:
Not sure what I was thinking here, this makes no sense, your interpretation is correct. Will adjust accordingly

Reply:
Done